docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-primary-registry.md

```diff
@@ -7,18 +7,20 @@ sidebar_position: 10
 tags: ["edge"]
 ---
 
-A primary registry is an in-cluster registry that stores images required for cluster deployment. Every disconnected Edge
-cluster must have a primary registry. Connected clusters can also benefit from using a primary registry for increased
-reliability and reduced bandwidth usage.
+A primary registry is an in-cluster registry that stores images required for cluster deployment. If your local cluster
+does not have connection to the internet, you can store all images required for cluster function in the primary registry
+so your cluster can operate in a local environment. In addition, both local and central clusters can benefit from using
+a primary registry for increased reliability and reduced bandwidth usage.
 
 :::preview
 
 :::
 
-When your Edge cluster is created for the first time, all images from add-on packs loaded from the content bundle or
-external registries are stored locally in the primary registry. Subsequent image pulls from the cluster are made to the
-primary registry. This allows your Edge cluster to reboot containers or add new nodes without being connected to the
-external network and reduce bandwidth usage.
+When your Edge cluster is created for the first time, all images in the `pack.content.images` field in each layer of the
+profile loaded from the content bundle or external registries are stored locally in the primary registry. Subsequent
```

Comment:
```suggestion
When your Edge cluster is created for the first time, all images in the `pack.content.images` field in each profile layer loaded from the content bundle or external registries are stored locally in the primary registry. Subsequent
```

Reply:
This sentence is quite long and hard to follow, although nothing is grammatically wrong with it.

Minor suggestion to shorten it slightly!